docs/configuration.md

```diff
@@ -74,8 +74,10 @@ such file with an updated version of the package).  It is safer to install an
     # install make into the minimal build chroot
     config_opts['chroot_additional_packages'] = 'make'
 
-You may also copy and edit an existing configuration file into a new one:
+You may also copy and edit an existing configuration file including the
+`default.cfg` into a new one:
 
+    $ cp /etc/mock/fedora-rawhide-x86_64.cfg ~/.config/mock/default.cfg
```

Comment:
This seems a bit confusing when it is "out of context".  Can you please create a separate paragraph describing `default.cfg` semantics (similar to the manpage)? And perhaps document _there_ how to define a per-user default.cfg in $HOME?

Btw., note that we plan to drop the default.cfg symlink by default #563.

Reply:
OK. I rebased the PR as your suggestion.